.github/CHANGELOG.md

```diff
@@ -1,11 +1,34 @@
-## Release 0.6.0a3 (development release)
+## Release 0.6.1a3 (development release)
```

Comment:
Could you remind me what the convention is for the blank development release template? Do we use the same version number as the current release (temporarily), or increment to say 0.7.0 until we finalize this before the next release?

Reply:
Yes, we basically use the same three version numbers as the current release temporarily and then bump it according to our rules before the next release.